.github/workflows/npm-publish.yaml

```diff
@@ -7,9 +7,8 @@ jobs:
     name: npm-publish
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v4
-    - name: Publish
-      run: yarn workspaces foreach --all -pt --since npm publish
+    - uses: actions/checkout@v4
+    - run: npm ci
+    - run: npm publish --workspaces --dry-run
```

Comment:
I left it as a dry-run for now, because I don't want to actually publish yet. Still just testing the `npm ci` and the `NODE_AUTH_TOKEN` work as intended.